app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java

```diff
@@ -146,8 +144,7 @@ public void setExternalVRContext(long externalContext) {
     }
 
     @Override
-    public void setFragmentManager(@NonNull FragmentManager fragmentManager, @NonNull ViewGroup container) {
-        mFragmentManager = fragmentManager;
+    public void setContainerView(@NonNull ViewGroup container) {
```

Comment:
Nitpicking a bit, but either the attribute name is wrong or the new name is wrong. If it's a container view then the attribute should be `mContainerView`, but if it's a view container then the method should be called `setViewContainer`

Reply:
Fixed. It's a container view that is acting as a container for views :) So I went with containerView, IMO it's a more logical name.